bitnami/mongodb/templates/networkpolicy.yaml

```
@@ -0,0 +1,46 @@
+{{- /*
+Copyright VMware, Inc.
+SPDX-License-Identifier: APACHE-2.0
+*/}}
+
+{{- if .Values.networkPolicy.enabled }}
+apiVersion: networking.k8s.io/v1
```

What if we use this common helper to set the apiVersion?
```suggestion
apiVersion: {{ include "common.capabilities.networkPolicy.apiVersion" . }}
```